vaadin-checkbox-flow-parent/vaadin-checkbox-flow-integration-tests/src/test/java/com/vaadin/flow/component/checkbox/tests/CheckboxIT.java

```diff
@@ -135,9 +135,11 @@ public void valueChangeCheckbox() {
     @Test
     public void accessibleCheckbox() {
         WebElement checkbox = layout.findElement(By.id("accessible-checkbox"));
+        WebElement inputElement = checkbox
+                .findElement(By.cssSelector("[slot=input]"));
         Assert.assertEquals(
                 "Accessible checkbox should have the aria-label attribute",
-                "Click me", checkbox.getAttribute("aria-label"));
+                "Click me", inputElement.getAttribute("aria-label"));
```

Comment:
Does it make sense to also test for `aria-labelledby`?

Reply:
We haven't been adding Integration tests for the `HasAriaLabel` functionality. This change was meant to fix an IT because `Checkbox` already had a `setAriaLabel` method.